easybuild/easyconfigs/p/PyCharm/PyCharm-2017.2.3.eb

```diff
@@ -0,0 +1,19 @@
+name = 'PyCharm'
+version = '2017.2.3'
+easyblock = 'Tarball'
```

Comment:
@shahzebsiddiqui Style nitpicking: `easyblock` line should be on top.

I've fixed this in https://github.com/shahzebsiddiqui/easybuild-easyconfigs/pull/1, if you merge that PR in your branch the PR here will be updated accordingly.